packages/flutter/lib/src/material/reorderable_list.dart

```diff
@@ -0,0 +1,545 @@
+      }
+      return child;
+    }
+
+    // We wrap the drag target in a Builder so that we can scroll to its specific context.
+    return new Builder(builder: (BuildContext context) {
+      return new DragTarget<Key>(
+        builder: buildDragTarget,
+        onWillAccept: (Key toAccept) {
+          setState(() {
+            _nextIndex = index;
+            _requestAnimationToNextIndex();
+          });
+          _scrollTo(context);
+          // If the target is not the original starting point, then we will accept the drop.
+          return _dragging == toAccept && toAccept != toWrap.key;
+        },
+        onAccept: (Key accepted) {},
+        onLeave: (Key leaving) {},
+      );
+    });
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    // We use the layout builder to constrain the cross-axis size of dragging child widgets.
+    return new LayoutBuilder(builder: (BuildContext context, BoxConstraints constraints) {
+        final List<Widget> wrappedChildren = <Widget>[];
+        if (widget.header != null) {
+          wrappedChildren.add(widget.header);
+        }
+        for (int i = 0; i < widget.children.length; i += 1) {
+          wrappedChildren.add(_wrap(widget.children[i], i, constraints));
+        }
+        const Key endWidgetKey = const Key('DraggableList - End Widget');
+        Widget finalDropArea;
+        switch (widget.scrollDirection) {
+          case Axis.horizontal:
+            finalDropArea = new SizedBox(
+              key: endWidgetKey,
+              width: _defaultDropAreaExtent,
+              height: constraints.maxHeight,
+            );
+            break;
+          case Axis.vertical:
+          default:
+            finalDropArea = new SizedBox(
+              key: endWidgetKey,
+              height: _defaultDropAreaExtent,
+              width: constraints.maxWidth,
+            );
+            break;
+        }
+        wrappedChildren.add(_wrap(
+          finalDropArea,
+          widget.children.length,
+          constraints),
+        );
+        return new SingleChildScrollView(
+          scrollDirection: widget.scrollDirection,
+          child: _buildContainerForScrollDirection(children: wrappedChildren),
+          padding: widget.padding, 
+          controller: _scrollController,
+        );
+    });
+  }
+}
+
+// Wrapper around the dragging feedback in a _ReorderableListContent.
+//
+// Calls onUpdateSize when the child's size changes.
+class _FeedbackWrapper extends StatefulWidget {
+  const _FeedbackWrapper({Key key, this.child, this.onUpdateSize}) : super(key: key);
+
+  final void Function(Size newSize) onUpdateSize;
+  final Widget child;
+
+  @override
+  _FeedbackWrapperState createState() => new _FeedbackWrapperState();
+}
+
+class _FeedbackWrapperState extends State<_FeedbackWrapper> {
+  @override
+  void initState() {
+    super.initState();
+    WidgetsBinding.instance.addPostFrameCallback(_computeSize);
+  }
+
+  @override
+  void dispose() {
```

Comment:
I don't think we want this widget at all, but this override clearly isn't needed.

Reply:
We discussed this offline, and agreed we can compute the size of the dragging widget when it's first picked.  

Now doing this with the global keys I added for semantics compatibility.